api/TimeAddressableMediaStore.yaml

```
@@ -1290,6 +1290,15 @@ paths:
           schema:
             default: false
             type: boolean
+        - name: accept_get_urls
+          in: query
+          description: |
+            A comma separated list of labels of flow segment `get_urls` to include in the response.
+            A 'default' label selects URLs with a 'default' or no label.
+            An empty `accept_get_urls` results in an empty or no `get_urls` flow segment property in the response.
```

Should we note, here in the description, the reason?
```suggestion
            An empty `accept_get_urls` results in an empty or no `get_urls` flow segment property in the response.
            Without `get_urls`, the response from the service is likely to be substantially faster.
```
?

---

I went with a slightly different wording in https://github.com/bbc/tams/pull/88/commits/d5ebe4a963071273b524439a55ceffc4241b94c3 because it depends on the number of flow segments and how optimised the pre-signed generation is.